.circleci/config.yml

```
@@ -136,7 +136,7 @@ jobs:
                   name: Test news-scripts
                   command: |
                       source venv/bin/activate
-                      mamba `find . -name *.spec.py` --format=documentation --enable-coverage
+                      mamba *spec.py --format=documentation --enable-coverage
```

```suggestion
                      mamba `find . -name "*.spec.py"` --format=documentation --enable-coverage
```
The find allows us to run all tests in the project, recursively.